docs/source/contributing.rst

```diff
@@ -35,58 +35,62 @@ Installing a Development Version
 --------------------------------
 
 
-.. todo::
+1. Fork both the SCICO repository and the SCICO Data submodule by clicking the Fork button on the `repository page <https://github.com/lanl/scico>`_. This will create a copy of both SCICO and SCICO Data repositories in your Git account.
```

Comment:
Perhaps refer to them as scico and scico-data? Does forking scico really automatically also for the submodule?

Reply:
That makes sense. I guess it was lost in the edits I made locally but it is supposed to have a link to the scico data submodule as well there to say that it will create a fork for each. You have to manually fork both. I am still looking for options to push changes upstream on the submodule which seems very tricky at the moment. But I have something I am currently testing locally on a smaller scale before trying it out on scico.